src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPCompletionProposal.java

```diff
@@ -155,12 +167,23 @@ private void updateCompletionItemFromResolved() {
         this.item = resolved;
     }
 
+    @Contract("null -> false")
```

Comment:
I added `@Contract` annotations here because the IDE's nullability inspection was having trouble determining the relationships between the params and the results and was raising false inspection positives.